### PR TITLE
Add WebGL1 Canvas Context

### PR DESCRIPTION
**Pull request by petele**
Replaces #1515

### CODE REVIEW

**Comment by captainbrosset** [comment]
I'm a bit unsure about the level of granularity here, and this is probably due to the fact that I don't know WebGL well enough. @petele @ddbeck @foolip what are your thoughts on the proposed features here?

For example, does "WebGL 32-bit floating-point color buffers" need to be its own feature? How about "Color management for WebGL"?

Are these sufficiently distinct from using WebGL? Do they allow developers to achieve a specific task? Or are they just part of how you use WebGL?

The concern I have with this is going too granular and departing from the notion of "features" which we want to be more relatable to developers than what BCD provides.

**Review by dji1admin — APPROVED** [review]

**Comment by petele** [comment]
@foolip, @ddbeck, and I have been going back and forth on these a whole bunch and have already merged a number of items into the main 'feature'. I get the concern about too many features, but the remaining features are each something that allows a specific functionality beyond the scope of general WebGL.

**Comment by captainbrosset** [comment]
Ok. Then I guess that's fine.

**Review by ddbeck — CHANGES_REQUESTED** [review]

File: features/webgl-color-buffer-float.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: WebGL 32-bit floating-point color buffers
```
[the remark this answers is not in the document]
Comment (petele) [reply features/webgl-color-buffer-float.yml]: done

**Review by ddbeck — APPROVED** [review]

**Pull request merged**